pages/_app.js

```diff
@@ -1,11 +1,21 @@
 import GlobalStyles from "../styles/GlobalStyles";
 import {useState} from "react";
+import {dbArray} from "../_db/dbArray";
 
 function MyApp({Component, pageProps}) {
+  /* Start: Logic for Login credential saving */
   const [loginName, setLoginName] = useState("Enter user name ... ");
   const onSubmitLogin = loginNameParam => {
     setLoginName(loginNameParam);
   };
+  /* End: Logic for Login credential saving */
```

Comment:
please rename this to `handleSubmitLogin` to follow our convention: The props should be named `on...`, the corresponding handlers should be named `handle...`